### ISSUE TITLE
Unable to build from git on babun (cygwin) - "can't exec autopoint"

### ISSUE THREAD

**Issue by ylupol** [allinurl/goaccess#1124]
Environment: Windows 10,  [babun](http://babun.github.io/) v1.2.0 (cygwin v1.7.35).

`$ autoreconf -fiv` fails with message:
``` zsh
{ goaccess } master » autoreconf -fiv                                                                               /C/Development/Tools/goaccess/goaccess
autoreconf-2.69: Entering directory `.'
autoreconf-2.69: running: autopoint --force
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 345.
autoreconf-2.69: failed to run autopoint: No such file or directory
autoreconf-2.69: autopoint is needed because this package uses Gettext
```

**Comment by allinurl** [comment]
Did you install `gettext` and/or `autopoint`?

**Comment by allinurl** [comment]
Running CYGWIN_NT-6.1  (v2.884 installer) I was able to install goaccess from git. Here're the packages I installed:

![cygwin-packages-goaccess](https://user-images.githubusercontent.com/5005367/40484616-d5dc9002-5f21-11e8-8613-9836032d4f13.jpg)

Do you have those installed in your machine?

**Comment by ylupol** [comment]
I noticed that Babun did not update `cygwin` for a long time. I updated it to the latest version manually like suggested [here](https://github.com/babun/babun/issues/516).

Then there was an issue while `autoreconf` execution with `perl.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory" (cygcrypt-0.dll not found)`. It was fixed like suggested [here](https://github.com/babun/babun/issues/800).

After that installation completed successfully.
Hope that helps.

@allinurl, thank you for the support!
